### DISCUSSION TITLE
Line resizing box

### DISCUSSION THREAD

**Discussion by svoboda200786** [fabricjs/fabric.js#7152] [Q&A]
I add a line by this code:

        let line = new fabric.Line([0, 0, 50, 50], {
            stroke: '#000',
            selectable: true,
            strokeWidth: '2',
        });
It looks like this:

https://disk.yandex.ru/i/6f2irCfTPJ_Kjw

As you see line resizing box is much biger than line itself. It's not good because when I click on space that should be free line also get a focus. For example in this code that draw an arrow: http://jsfiddle.net/ug2gskj1/  you can change angle of arrow by moving triangle or circle only untill you click somewhere else. Then if you click on triangle or circle one more time and try to move them to change the angle, they would not get focus and the arrow would move as a whole.

Is there any way to make resizing box smaller? So start of the line would be in one conner of the box, and end in another. And what are units of coordinates? fabric.Line([0, 0, 50, 50]) - as I see it's not pixels. Percents?

**Comment by svoboda200786** [comment]
I found the solution - code on jsfiddle contains value of stroke width in quotes - strokeWidth: '2'. I deleted quotes and all work ok:-)

**Reply by asturur** [reply]
I will double check that in the docs strokeWidth is defined as Number
